scripts/build_deb.sh

```diff
@@ -20,8 +20,10 @@ if [ "$EUID" != "0" ]; then
 fi
 
 OS=linux
-ARCH=$1
+ARCH="$1"
 OUTDIR="$2"
+CHANNEL=${CHANNEL:-$3}
+PKG_NAME=$(./scripts/compute_package_name.sh "${CHANNEL:-stable}")
```

Comment:
Do we want to default to stable? It could be unfortunate to accidentally package a beta release as stable

Reply:
It *shouldn't* ever have an empty value by the time line 26 is parsed.  Is this 12 of one half dozen of the other?

Reply:
I tried googling that idiom and I'm still confused. But I like the confidence so I'm ok with this default

Reply:
https://idioms.thefreedictionary.com/six+of+one+and+half+a+dozen+of+the+other

I can change the default if you want.